refactor TOC and samples page

laujan opened the pull request.

opbld34 commented:
Docs Build status updates of commit _[0c90849](https://github.com/MicrosoftDocs/msteams-docs/commits/0c90849d2f41ce5efe1d04f42c6e6ae6073803f2)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[.openpublishing.redirection.json](https://github.com/MicrosoftDocs/msteams-docs/blob/update-toc-and-samples/.openpublishing.redirection.json) | :white_check_mark:Succeeded | |
[msteams-platform/TOC.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/update-toc-and-samples/msteams-platform/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoftteams/platform/tutorials/code-samples?branch=pr-en-us-1559) |
[msteams-platform/tutorials/code-samples.md](https://github.com/MicrosoftDocs/msteams-docs/blob/update-toc-and-samples/msteams-platform/tutorials/code-samples.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoftteams/platform/tutorials/code-samples?branch=pr-en-us-1559) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C4%5C17%5Cd98cc23d-94e8-765b-0855-bdd8964dc537%5CPullRequest%5C202004170011521695-1559%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=kMMkJCrT13%2B2YEc%2F8icj0ypPo4F%2BiogFCNgZSekZ1ps%3D&st=2020-04-17T00%3A08%3A23Z&se=2020-05-18T00%3A13%3A23Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

opbld33 commented:
Docs Build status updates of commit _[8c2692a](https://github.com/MicrosoftDocs/msteams-docs/commits/8c2692a8c4f7916173a0ea6fbb32ec07b05f3e61)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[.openpublishing.redirection.json](https://github.com/MicrosoftDocs/msteams-docs/blob/update-toc-and-samples/.openpublishing.redirection.json) | :white_check_mark:Succeeded | |
[msteams-platform/TOC.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/update-toc-and-samples/msteams-platform/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoftteams/platform/tutorials/code-samples?branch=pr-en-us-1559) |
[msteams-platform/tutorials/code-samples.md](https://github.com/MicrosoftDocs/msteams-docs/blob/update-toc-and-samples/msteams-platform/tutorials/code-samples.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoftteams/platform/tutorials/code-samples?branch=pr-en-us-1559) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C4%5C17%5Cd98cc23d-94e8-765b-0855-bdd8964dc537%5CPullRequest%5C202004171804329695-1559%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=QZsD2HKnWgdLmRNDi91lQ4uSrgj2POurMMocU2KzZ9I%3D&st=2020-04-17T18%3A01%3A13Z&se=2020-05-18T18%3A06%3A13Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

opbld32 commented:
Docs Build status updates of commit _[eead9f4](https://github.com/MicrosoftDocs/msteams-docs/commits/eead9f43e26b5d8308904aaad29fdb9da90bd831)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[.openpublishing.redirection.json](https://github.com/MicrosoftDocs/msteams-docs/blob/update-toc-and-samples/.openpublishing.redirection.json) | :white_check_mark:Succeeded | |
[msteams-platform/TOC.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/update-toc-and-samples/msteams-platform/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoftteams/platform/tutorials/code-samples?branch=pr-en-us-1559) |
[msteams-platform/tutorials/code-samples.md](https://github.com/MicrosoftDocs/msteams-docs/blob/update-toc-and-samples/msteams-platform/tutorials/code-samples.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoftteams/platform/tutorials/code-samples?branch=pr-en-us-1559) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C4%5C17%5Cd98cc23d-94e8-765b-0855-bdd8964dc537%5CPullRequest%5C202004171823487139-1559%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=IUTBIh%2FlIAUT9mZ1FB%2FpW%2BA7PgT5eoobsX4qWEAqx9w%3D&st=2020-04-17T18%3A20%3A23Z&se=2020-05-18T18%3A25%3A23Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

opbld30 commented:
Docs Build status updates of commit _[6bf2ca6](https://github.com/MicrosoftDocs/msteams-docs/commits/6bf2ca6ee2da906800990a2443c6455bc2efe407)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[.openpublishing.redirection.json](https://github.com/MicrosoftDocs/msteams-docs/blob/update-toc-and-samples/.openpublishing.redirection.json) | :white_check_mark:Succeeded | |
[msteams-platform/TOC.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/update-toc-and-samples/msteams-platform/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoftteams/platform/tutorials/code-samples?branch=pr-en-us-1559) |
[msteams-platform/tutorials/code-samples.md](https://github.com/MicrosoftDocs/msteams-docs/blob/update-toc-and-samples/msteams-platform/tutorials/code-samples.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoftteams/platform/tutorials/code-samples?branch=pr-en-us-1559) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C4%5C17%5Cd98cc23d-94e8-765b-0855-bdd8964dc537%5CPullRequest%5C202004171828380368-1559%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=pFQf9%2FPbVAk4cE5JgimHA28hefwOddPpY6qk%2BKTp2gM%3D&st=2020-04-17T18%3A25%3A08Z&se=2020-05-18T18%3A30%3A08Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

clearab commented on msteams-platform/tutorials/code-samples.md:
```diff
@@ -3,11 +3,21 @@ title: Microsoft Teams code samples
+# Tutorials and code samples for the Microsoft Teams developer platform
 
-Here you will find a list of code samples that demonstrate various capabilities of the Microsoft Teams development platform and how to build apps to leverage those features.
+Here you will find a list of tutorials and code samples that demonstrate how you can extend the Teams developer platform capabilities by creating custom apps.
```
Not a big fan of "developer platform" here - they aren't extending the developer platform, they are using the developer platform to extend Teams.

clearab commented on msteams-platform/tutorials/code-samples.md:
```diff
@@ -3,11 +3,21 @@ title: Microsoft Teams code samples
+| Task modules |  [Collect input in Microsoft Teams with Task Modules](https://docs.microsoft.com/learn/modules/msteams-task-modules/) |
+| Conversational bots  | [Create interactive conversational bots for Microsoft Teams](https://docs.microsoft.com/learn/modules/msteams-conversation-bots/)  |
+
+## Getting started with code samples
```
Should probably add links to the Python samples in the Messaging Extensions section.

clearab replied:
Also add a link to the all-up bot samples page in the intro of the bot section: https://github.com/Microsoft/BotBuilder-Samples

laujan commented:
Hi @aosolis. This pull request addresses GitHub issue #1511. I believe the Hello World samples are too outdated to be helpful and I wanted to incorporate the Learn modules into our platform so I updated the samples page. Also, I wanted to separate the production-ready templates/integrations from the samples/tutorials so I refactored the TOC. I look forward to your feedback. Thank you!

clearab commented on msteams-platform/TOC.yml:
```diff
@@ -5,22 +5,22 @@
   - name: Production-ready app templates
     href: samples/app-templates.md
-  - name: Production-ready Shifts Connectors
+  - name: Production-ready Shifts connectors
```
I believe "Connectors" here is a product name and should be capitalized, but I'm not 100%

opbld31 commented:
Docs Build status updates of commit _[6bf2ca6](https://github.com/MicrosoftDocs/msteams-docs/commits/6bf2ca6ee2da906800990a2443c6455bc2efe407)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[.openpublishing.redirection.json](https://github.com/MicrosoftDocs/msteams-docs/blob/update-toc-and-samples/.openpublishing.redirection.json) | :white_check_mark:Succeeded | |
[msteams-platform/TOC.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/update-toc-and-samples/msteams-platform/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoftteams/platform/tutorials/code-samples?branch=pr-en-us-1559) |
[msteams-platform/tutorials/code-samples.md](https://github.com/MicrosoftDocs/msteams-docs/blob/update-toc-and-samples/msteams-platform/tutorials/code-samples.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoftteams/platform/tutorials/code-samples?branch=pr-en-us-1559) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C4%5C17%5Cd98cc23d-94e8-765b-0855-bdd8964dc537%5CPullRequest%5C202004171942571270-1559%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=A3KS4UAKOJCXf%2BRsfsshwCZlHLrCLHxFzPUKK791fjU%3D&st=2020-04-17T19%3A39%3A54Z&se=2020-05-18T19%3A44%3A54Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

laujan commented:
Thanks, Andrew! Hope all is well with you and your family! I’m adjusting to the “new normal” but I miss the office comradery. 🙂

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

From: Andrew Clear<mailto:notifications@github.com>
Sent: Friday, April 17, 2020 12:44 PM
To: MicrosoftDocs/msteams-docs<mailto:msteams-docs@noreply.github.com>
Cc: la januari<mailto:la.januari@microsoft.com>; State change<mailto:state_change@noreply.github.com>
Subject: Re: [MicrosoftDocs/msteams-docs] refactor TOC and samples page (#1559)


@clearab commented on this pull request.

All-up I like this, a nice clean landing page for everything!


In msteams-platform/tutorials/code-samples.md<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2Fmsteams-docs%2Fpull%2F1559%23discussion_r410432476&data=02%7C01%7Cla.januari%40microsoft.com%7C53d99804467e44bb424b08d7e307ae3e%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637227494438590610&sdata=Onk5kZJlRxJ5RdeaR8unS6pbOGP7ROCN0Qx96aypxXU%3D&reserved=0>:

>



-Here you will find a list of code samples that demonstrate various capabilities of the Microsoft Teams development platform and how to build apps to leverage those features.



+Here you will find a list of tutorials and code samples that demonstrate how you can extend the Teams developer platform capabilities by creating custom apps.



Not a big fan of "developer platform" here - they aren't extending the developer platform, they are using the developer platform to extend Teams.


In msteams-platform/tutorials/code-samples.md<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2Fmsteams-docs%2Fpull%2F1559%23discussion_r410432834&data=02%7C01%7Cla.januari%40microsoft.com%7C53d99804467e44bb424b08d7e307ae3e%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637227494438590610&sdata=%2FxAwRIIQ0vhEPP%2FkoA4PsWGoF3plPozU65Db6VeGYd8%3D&reserved=0>:

>



-## Getting samples



+## Getting started with learning modules



+



+| Capability| Learning Module|



+|--------|-------------|



+| Tabs  — embedded web experiences  |  [Create embedded web experiences with tabs for Microsoft Teams](https://docs.microsoft.com/learn/modules/embedded-web-experiences/) |



+| Webhooks and connectors  |  [Connect web services to Microsoft Teams with webhooks and Office 365 Connectors](https://docs.microsoft.com/learn/modules/msteams-webhooks-connectors/) |



+|Messaging extensions  | [Task-oriented interactions in Microsoft Teams with messaging extensions](https://docs.microsoft.com/learn/modules/msteams-messaging-extensions/)  |



+| Task modules |  [Collect input in Microsoft Teams with Task Modules](https://docs.microsoft.com/learn/modules/msteams-task-modules/) |



+| Conversational bots  | [Create interactive conversational bots for Microsoft Teams](https://docs.microsoft.com/learn/modules/msteams-conversation-bots/)  |



+



+## Getting started with code samples



Should probably add links to the Python samples in the Messaging Extensions section.


In msteams-platform/tutorials/code-samples.md<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2Fmsteams-docs%2Fpull%2F1559%23discussion_r410433313&data=02%7C01%7Cla.januari%40microsoft.com%7C53d99804467e44bb424b08d7e307ae3e%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637227494438600601&sdata=RkiReio%2FfuqAbf2N3W%2Ftu0tbfIAwLanISdxql27LM4M%3D&reserved=0>:

>



-## Getting samples



+## Getting started with learning modules



+



+| Capability| Learning Module|



+|--------|-------------|



+| Tabs  — embedded web experiences  |  [Create embedded web experiences with tabs for Microsoft Teams](https://docs.microsoft.com/learn/modules/embedded-web-experiences/) |



+| Webhooks and connectors  |  [Connect web services to Microsoft Teams with webhooks and Office 365 Connectors](https://docs.microsoft.com/learn/modules/msteams-webhooks-connectors/) |



+|Messaging extensions  | [Task-oriented interactions in Microsoft Teams with messaging extensions](https://docs.microsoft.com/learn/modules/msteams-messaging-extensions/)  |



+| Task modules |  [Collect input in Microsoft Teams with Task Modules](https://docs.microsoft.com/learn/modules/msteams-task-modules/) |



+| Conversational bots  | [Create interactive conversational bots for Microsoft Teams](https://docs.microsoft.com/learn/modules/msteams-conversation-bots/)  |



+



+## Getting started with code samples



Also add a link to the all-up bot samples page in the intro of the bot section: https://github.com/Microsoft/BotBuilder-Samples<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FMicrosoft%2FBotBuilder-Samples&data=02%7C01%7Cla.januari%40microsoft.com%7C53d99804467e44bb424b08d7e307ae3e%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637227494438600601&sdata=qoBa9deK%2FZxyg2nSYuK8kbHohkJM8KWb%2F9BRSZF17R4%3D&reserved=0>

—
You are receiving this because you modified the open/close state.
Reply to this email directly, view it on GitHub<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2Fmsteams-docs%2Fpull%2F1559%23pullrequestreview-395725170&data=02%7C01%7Cla.januari%40microsoft.com%7C53d99804467e44bb424b08d7e307ae3e%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637227494438610606&sdata=wQJ%2FjUR%2FvIlLq1q54Zpu0nMBUcGYWyF7dtDCmzdNKx8%3D&reserved=0>, or unsubscribe<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAMBJMSW3HJEM7GLV65STIWTRNCWQDANCNFSM4MKKSQMA&data=02%7C01%7Cla.januari%40microsoft.com%7C53d99804467e44bb424b08d7e307ae3e%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637227494438610606&sdata=PsKiNelsPPYUhbVPyQox20jq%2F%2BtMDmWZfuc%2F0j99xBs%3D&reserved=0>.

opbld32 commented:
Docs Build status updates of commit _[1d54bbd](https://github.com/MicrosoftDocs/msteams-docs/commits/1d54bbdb8f9b68a2cecb8487d1081a9a614bdd18)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[.openpublishing.redirection.json](https://github.com/MicrosoftDocs/msteams-docs/blob/update-toc-and-samples/.openpublishing.redirection.json) | :white_check_mark:Succeeded | |
[msteams-platform/TOC.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/update-toc-and-samples/msteams-platform/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoftteams/platform/tutorials/code-samples?branch=pr-en-us-1559) |
[msteams-platform/tutorials/code-samples.md](https://github.com/MicrosoftDocs/msteams-docs/blob/update-toc-and-samples/msteams-platform/tutorials/code-samples.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoftteams/platform/tutorials/code-samples?branch=pr-en-us-1559) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C4%5C17%5Cd98cc23d-94e8-765b-0855-bdd8964dc537%5CPullRequest%5C202004172009174595-1559%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=itEedDN30eTWWpMDcZSd06vLyd9DmNnmbpc28XBj1eQ%3D&st=2020-04-17T20%3A05%3A57Z&se=2020-05-18T20%3A10%3A57Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

opbld32 commented:
Docs Build status updates of commit _[8a905fd](https://github.com/MicrosoftDocs/msteams-docs/commits/8a905fde0aa1e0979d7c2925ef8fda5eb705e35a)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[.openpublishing.redirection.json](https://github.com/MicrosoftDocs/msteams-docs/blob/update-toc-and-samples/.openpublishing.redirection.json) | :white_check_mark:Succeeded | |
[msteams-platform/TOC.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/update-toc-and-samples/msteams-platform/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoftteams/platform/tutorials/code-samples?branch=pr-en-us-1559) |
[msteams-platform/tutorials/code-samples.md](https://github.com/MicrosoftDocs/msteams-docs/blob/update-toc-and-samples/msteams-platform/tutorials/code-samples.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoftteams/platform/tutorials/code-samples?branch=pr-en-us-1559) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C4%5C27%5Cd98cc23d-94e8-765b-0855-bdd8964dc537%5CPullRequest%5C202004272035322922-1559%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=u0zd1T626kE6c5QYjYkv9s%2FKw8zqSTzyJUwjZfTYb4Q%3D&st=2020-04-27T20%3A32%3A12Z&se=2020-05-28T20%3A37%3A12Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

opbld34 commented:
Docs Build status updates of commit _[a3924de](https://github.com/MicrosoftDocs/msteams-docs/commits/a3924de4aa4624bb137ec3657e0eb37400996c97)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[.openpublishing.redirection.json](https://github.com/MicrosoftDocs/msteams-docs/blob/update-toc-and-samples/.openpublishing.redirection.json) | :white_check_mark:Succeeded | |
[msteams-platform/TOC.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/update-toc-and-samples/msteams-platform/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoftteams/platform/tutorials/code-samples?branch=pr-en-us-1559) |
[msteams-platform/tutorials/code-samples.md](https://github.com/MicrosoftDocs/msteams-docs/blob/update-toc-and-samples/msteams-platform/tutorials/code-samples.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/microsoftteams/platform/tutorials/code-samples?branch=pr-en-us-1559) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C4%5C27%5Cd98cc23d-94e8-765b-0855-bdd8964dc537%5CPullRequest%5C202004272327252576-1559%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=IOmgJJZ8%2FI7P8pEZqa40KOW9Cx0js5ikaCkRPQZUShA%3D&st=2020-04-27T23%3A23%3A58Z&se=2020-05-28T23%3A28%3A58Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

laujan commented:
closes #1511
